examples/cfd/lid_driven_cavity_2d.py

```diff
@@ -49,7 +50,7 @@ def define_boundary_indices(self):
 
     def setup_boundary_conditions(self):
         lid, walls = self.define_boundary_indices()
-        bc_top = EquilibriumBC(rho=1.0, u=(0.02, 0.0), indices=lid)
+        bc_top = EquilibriumBC(rho=1.0, u=(self.prescribed_vel, 0.0), indices=lid)
```

Comment:
Same comment for EquilibriumBC in all examples.

Reply:
I did not do this. Eq BC is more tricky, as you need to set profile for both rho and u. I have to check the literature on this first to ensure this is done correctly.